.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
```

Comment:
I understand the npm updates, but what will this do?

Reply:
This will update the used actions in all workflows we have in this repo

`actions/setup-node` is currently at v2 for instance
https://github.com/styfle/cancel-workflow-action/blob/3079606bd087920c96acfa9ca9f806fdd0cc3afd/.github/workflows/cancel-self.yml#L16

Reply:
How does it know its safe? I didn't think github actions followed semver? 

For example, will v1 here automatically request an update to v2?

Reply:
Actions are just like any other packages and should use semver as a good practice

I learned about being able to update actions version on https://docs.github.com/en/code-security/supply-chain-security/keeping-your-actions-up-to-date-with-dependabot